tex/gregoriotex-signs.tex

```diff
@@ -1141,34 +1141,34 @@
   \relax %
 }%
 
-\newif\ifgre@hepisemusbridge
-\gre@hepisemusbridgetrue
+\newif\ifgre@hepisemabridge
+\gre@hepisemabridgetrue
 
-\def\AddHEpisemusBridges{%
-  \gre@deprecated{\protect\AddHEpisemusBridges}{\protect\gresethepisemus{bridge}}%
-  \gre@hepisemusbridgetrue%
+\def\AddHEpisemaBridges{%
```

Comment:
The names of deprecated functions should not be changed as the purpose of having them still in the code base is to maintain backwards compatibility.